packages/mockyeah-cli/lib/boot.js

```diff
@@ -33,4 +67,8 @@ module.exports = function boot(callback) {
 
     callback.call(this, env);
   });
-};
+}
+
+boot.checkVersionMatchWithPackage = checkVersionMatchWithPackage;
```

Comment:
Should we externalize `checkVersionMatchWithPackage` into lib? I get a bad smell when methods are exported just for testing.

Reply:
@ryanricard Probably. Can refactor. Hold up.

Reply:
@ryanricard OK, refactored to external file in baaa05a.